recipe/meta.yaml

```diff
@@ -0,0 +1,182 @@
+{% set name = 'hail' %}
+{% set version = '0.2.127' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # the build process requires a repository to set some variables
+  # (e.g. version)
+  git_url: https://github.com/hail-is/hail.git
+  git_rev: {{ version }}
+  patches:
+    - patches/0001-remove-vendored-libsimdpp.patch
+    - patches/0002-remove-vendored-catch2.patch
+    - patches/0003-patch-requirements.patch
+    - patches/0004-pluggy-old-style-wrappers.patch
+    - patches/0005-patch-spark-3.3.2-to-3.4.1.patch
+
+build:
+  number: 0
+  # avro and others a not available for win
+  # openjdk and pyspark not available for s390x
+  skip: true  # [py<39 or win or py>311 or s390x]
+  skip: true  # [blas_impl == "mkl"]
+  entry_points:
+    - hailctl = hailtop.hailctl.__main__:main
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make      # [linux]
+    - patch     # [not win]
+    - m2-patch  # [win]
+    - git
+  host:
+    - python
+    - pip
+    - setuptools
+    # prerequisites
+    - pytest-runner
+    - pyspark >=3.3.2,<=3.4.1
+    - openjdk 11.*
+    - lz4
+    # blas (lapack is brought in along)
+    - openblas-devel {{ openblas }}  # [blas_impl == "openblas"]
```

Comment:
> I ran a couple of the examples in the docs (including the BlockMatrix examples which use blas+lapack) and it seems to work.

Tests that before were failing because of missing BLAS / LAPACK, now succeed so it's in line with your conclusions. It was indeed Scala in a .jar (the JVM) looking for those two. For some reason, the JVM can find them.

We will keep this in mind in case we will run into troubles, unless anyone has a better suggestion.

Reply:
(thank you for the details: I bookmarked your comment for future use)

Reply:
Can you do me one last favor @lorepirri? Can you remove
`- openblas-devel {{ openblas }}  # [blas_impl == "openblas"]`
and see if indeed it still builds? If it does, I would simply remove this line

Reply:
```suggestion
```

Reply:
See also this thread: https://anaconda.slack.com/archives/C04QPV5GWBW/p1710338065923809